source/contracts/reporting/Universe.sol

```diff
@@ -384,31 +384,9 @@ contract Universe is DelegationTarget, ITyped, Initializable, IUniverse {
         return _newValue;
     }
 
-    function getOrCacheReportingFeeDivisor() public onlyInGoodTimes returns (uint256) {
-        IFeeWindow _feeWindow = getOrCreateCurrentFeeWindow();
-        IFeeWindow _previousFeeWindow = getOrCreatePreviousFeeWindow();
-        uint256 _currentFeeDivisor = shareSettlementFeeDivisor[_feeWindow];
-        if (_currentFeeDivisor != 0) {
-            return _currentFeeDivisor;
-        }
-        uint256 _repMarketCapInAttoeth = getRepMarketCapInAttoeth();
-        uint256 _targetRepMarketCapInAttoeth = getTargetRepMarketCapInAttoeth();
-        uint256 _previousFeeDivisor = shareSettlementFeeDivisor[_previousFeeWindow];
-        if (_previousFeeDivisor == 0) {
-            _previousFeeDivisor = Reporting.getDefaultReportingFeeDivisor();
-        }
-        if (_targetRepMarketCapInAttoeth == 0) {
-            _currentFeeDivisor = Reporting.getMaximumReportingFeeDivisor();
-        } else {
-            _currentFeeDivisor = _previousFeeDivisor.mul(_repMarketCapInAttoeth).div(_targetRepMarketCapInAttoeth);
-        }
-
-        _currentFeeDivisor = _currentFeeDivisor
-            .max(Reporting.getMinimumReportingFeeDivisor())
-            .min(Reporting.getMaximumReportingFeeDivisor());
-
-        shareSettlementFeeDivisor[_feeWindow] = _currentFeeDivisor;
-        return _currentFeeDivisor;
+    // Hardcode the reporting fee until we implement the auction feature or some other decentralized REP price oracle
+    function getOrCacheReportingFeeDivisor() public view returns (uint256) {
+        return Reporting.getMaximumReportingFeeDivisor();
```

Comment:
I have been thinking about this, I wonder if we should bump the min reporting fee to 0.1% or 1% now that we are hard-coding it?  This will let us run safely for longer before we _need_ to upgrade.

Reply:
This is a good point. The default fee actually starts at 1% in the old code, so maybe we could go with that. Should probably be discussed more in chat